File: src/com/fenghuo/notes/context/NoteEvent.java
package com.fenghuo.notes.context;

/**
 * Created by gang on 16-7-10.
 */
public class NoteEvent {

    public static final int TPYE_MENU_CLICK = 0;
    public static final int TPYE_UPDATE_MENU_STATE = 1;

    public int what;
    public int argInt1;
    public int argInt2;
    public boolean argBoolean;
    public Object object;

    public NoteEvent(int what) {
        this.what = what;
    }
}

File: src/com/fenghuo/notes/context/NoteEvent.kt
package com.fenghuo.notes.context

/**
 * Created by gang on 16-7-10.
 */
class NoteEvent(var what: Int) {
    var argInt1: Int = 0
    var argInt2: Int = 0
    var argBoolean: Boolean = false
    var `object`: Any? = null

    companion object {

        val TPYE_MENU_CLICK = 0
        val TPYE_UPDATE_MENU_STATE = 1
    }
}